modules/scanning-images.adoc

```diff
@@ -156,3 +120,40 @@ The following vulnerability sources are not updated by the vendor:
 * Scanner does not support the Fedora operating system because Fedora does not maintain a vulnerability database.
```

Comment:
```suggestion
Scanner does not support the Fedora operating system because Fedora does not maintain a vulnerability database.
```
Do you mind making this change while fixing the others? Not sure why this is bulleted.